### ISSUE TITLE
double log entries, and no bans

### ISSUE THREAD

**Issue by joshp23** [SweBarre/log-login#9]
log-login seems to be double logging every failed attempt. Perhaps related to [this](log-login seems to be double logging every failed attempt. Perhaps related to  #8 logging false positives

these numerous entries are also not resulting in bans.

**Comment by r1bnc** [comment]
I agree with you. mine has not just double but 4x entries and always without the username, it just like when someone visit the site and it logs them as failed logons.
I added a conditional statement if the username is empty then do not log it: 
plugins.php 
```
 if (strlen($_REQUEST['username']) > 0) { barre_log_login_log2file( 'LOGIN FAILURE '. $_REQUEST['username'] );
    }

```
  for the banning I modified the system time to local and added a timezone of the fail2ban to UTC.
